### ISSUE TITLE
Version in docs incorrect

### ISSUE THREAD

**Issue by tmillenaar** [tmillenaar/GridKit#28]
The docs on github pages refer to an incorrect version of GridKit (0.1.0.beta).
It should refer to the most recent release the docs are reflecting.

This is a problem related to the version in version.py.
This version is updated when the publish to pypi workflow is ran in the CICD. The line that replaces the version number to that of the tag is:
`run: sed -i -r "s|\"[[:alnum:]]+\.[[:alnum:]]+\.[[:alnum:]]+.+|\"${GITHUB_REF/refs\/tags\//}\"|" ./gridkit/version.py; cat ./gridkit/version.py`

We need to run a similar line in documentation.yaml such that the version number in the docs accurately represents the current version.

**Comment by tmillenaar** [comment]
It does look like the sed command is successfully executed after the previous doc build, but it is not correct in the build docs somehow. From https://github.com/tmillenaar/GridKit/actions/runs/6067143264/job/16458462991 :
![image](https://github.com/tmillenaar/GridKit/assets/45204355/535bdd16-fca3-49fd-b6e4-0568a98dcbf2)
![image](https://github.com/tmillenaar/GridKit/assets/45204355/5e1fa822-20cc-499a-90f8-cc03aea4fd65)

**Comment by tmillenaar** [comment]
The version has been fixed in the recent release where the CICD deployment process was overhauled
![image](https://github.com/tmillenaar/GridKit/assets/45204355/bff212c0-d9d2-43c6-af57-18cab6aff865)
The problem was related to the command `pip install ./[doc]`. I needed to call `pip install -e ./[doc]`.
In the former case the package was installed on the system and that was used. I then updated the local file but not the recently installed files. With the `-e` flag I now make sure not to install the package on the system and use the files in the local folder instead (they will be linked) so now the modified files are used when building the docs.
